docs/dev/guidelines/client-tests.rst

```diff
@@ -0,0 +1,257 @@
+******
+Client Tests
+******
```

Comment:
I don't know if this is really important, VSC is complaining about the amount of * :D. After adding it, it still is complaining ((INFO/1) Enumerated list start value not ordinal-1: "2" (ordinal 2)), but maybe this can be ignored...
```suggestion
************
Client Tests
************
```

Reply:
Oh, now I see. I didn't know that the asterics should cover the whole title.
Thank you 😊